spirv_glsl.cpp

```diff
@@ -5651,9 +5654,10 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 	case OpVectorTimesMatrix:
 	case OpMatrixTimesVector:
 	{
-		// If the matrix needs transpose, just flip the multiply order.
+		// If the matrix needs transpose and it is square, just flip the multiply order.
+		SPIRType *t;
```

Comment:
Should probably just assign t here, I feel it's a bit sketchky to assign in the branch.

Reply:
Done.